### PR TITLE
Onboarding script

### PR DESCRIPTION
**Pull request by garrettr**
Write a script (like a movie script, not a shell script) for FPF staff to follow in onboarding new users onto the Redmine-based support site.

### CODE REVIEW

**Comment by garrettr** [comment]
Rebased on top of current master (needed after merging #4).

**Comment by garrettr** [comment]
@conorsch @ageis If you want to review this, you can easily see the HTML (with TOC) here: https://rawgit.com/freedomofpress/redmine_docs/onboarding-script/onboarding_issue_script.html

GitHub renders .org automatically, but it doesn't do the TOC, which makes it a little harder to review IMHO.

**Comment by conorsch** [comment]
@garrettr Thorough and explicit! No changes to the content, I'm happy with it as-is for use with our beta testers.

A note on the beta-testers: let's force one to use the encrypted email workflow and the other to use the web-based workflow. That should help in testing both the actual ticketing setup and the onboarding script.

File: onboarding_issue_script.html [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+On the original "Onboarding" issue, write the following reply:
+</p>
+
+<p>
+<b>Reply</b>:
+</p>
+
+<div class="org-src-container">
+
+<pre class="src src-markdown">Great! Now that you're comfortable creating and replying to issues, you're ready to use Redmine to report issues and request support for your Redmine instance.
```
Comment (ageis) [inline onboarding_issue_script.html]: s/Redmine instance/SecureDrop instance

**Comment by ageis** [comment]
@garrettr Wow, nice job! lgtm

File: onboarding_issue_script.html (hunk 1 above)
Comment (garrettr) [inline onboarding_issue_script.html]: Good catch

**Pull request merged**